python/probe_screen.py

```diff
@@ -2027,7 +2070,9 @@ def on_spbtn_block_height_value_changed( self, gtkspinbutton, data = None ):
             self.buffer.delete(i,self.buffer.get_end_iter())
         i.set_line(0)
         self.buffer.insert(i, "%s \n" % c)
-    def clicked_btn_probe_tool_setter(self, data = None):
+
+    @restore_mode
+    def clicked_btn_probe_tool_setter(self, gtkbuttondata = None):
```

Comment:
i believe this is a typo @kiall, should be: `def clicked_btn_probe_tool_setter(self, gtkbutton, data = None):`

Reply:
Fixed in https://github.com/linuxcnc-probe-screen/probe-screen - a hopefully community maintained fork :)

Reply:
awesome! thank you for the fixes and the fork! by the way, i tested all these changes and everything seems to work now